receiver/carbonreceiver/transport/client/plaintext_client.go

```diff
@@ -52,8 +51,8 @@ const defaultTimeout = 5
 // https://github.com/census-ecosystem/opencensus-go-exporter-graphite/tree/master/internal/client
 // and modified for the needs of testing the Carbon receiver package and is not
 // intended/tested to be used in production.
-func NewGraphite(transport Transport, host string, port int) (*Graphite, error) {
-	graphite := &Graphite{Host: host, Port: port}
+func NewGraphite(transport Transport, endpoint string) (*Graphite, error) {
```

Comment:
Is this function being used anywhere?  I expected to see some changes other than tests where endpoint is passed into this function instead of host and port.

Reply:
Just tests :)) see that CI passed :))

Reply:
So I don't think this is my fault :))